src/notification.c

```diff
@@ -227,8 +227,10 @@ handle_add_notification (XdpImplNotification *object,
                          const gchar *arg_id,
                          GVariant *arg_notification)
 {
-  if (gtk_notifications == NULL ||
-      g_dbus_proxy_get_name_owner (G_DBUS_PROXY (gtk_notifications)) == NULL ||
+  g_autofree char* owner = gtk_notifications ?
```

Comment:
Minor style nitpick, not a blocker:

```suggestion
  g_autofree char* owner = gtk_notifications != NULL ?
```